src/services/rsvp/rsvp-router.ts

```diff
@@ -0,0 +1,181 @@
+ *      "status": "ACCEPTED",
+ *      "response": "ACCEPTED",
+ *      "reviewer": "reviewer1",
+ *      "emailSent": true,
+ * 	}
+ *
+ * @apiUse strongVerifyErrors
+ */
+rsvpRouter.get("/", strongJwtVerification, async (_: Request, res: Response) => {
+    const payload: JwtPayload = res.locals.payload as JwtPayload;
+
+    const userId: string = payload.id;
+
+    const queryResult: DecisionInfo | null = await Models.DecisionInfo.findOne({ userId: userId });
+
+    //Returns error if query is empty
+    if (!queryResult) {
+        return res.status(StatusCode.ClientErrorBadRequest).send({ error: "UserNotFound" });
+    }
+
+    const toReturn = queryResult.toObject();
+
+    //Filters data if caller doesn't have elevated perms
+    if (!hasElevatedPerms(payload)) {
+        delete toReturn.reviewer;
+        delete toReturn.emailSent;
+    }
+
+    return res.status(StatusCode.SuccessOK).send(toReturn);
+});
+
+/**
+ * @api {put} /rsvp/ Put /rsvp/
+ * @apiGroup rsvp
+ * @apiDescription Updates an rsvp for the currently authenticated user (determined by the JWT in the Authorization header).
+ *
+ * @apiBody {boolean} isAttending Whether or whether not the currently authenticated user is attending
+ * @apiParamExample {json} Example Request:
+ * {
+ *      "isAttending": false
+ * }
+ *
+ * @apiSuccess (200: Success) {string} MongoDB object ID
+ * @apiSuccess (200: Success) {string} userId
+ * @apiSuccess (200: Success) {string} User's applicatoin status
+ * @apiSuccess (200: Success) {string} User's Response (whether or whether not they're attending)
+ * @apiSuccess (200: Success) {string} Reviwer
+ * @apiSuccess (200: Success) {boolean} Whether email has been sent
+ * @apiSuccessExample Example Success Response:
+ * 	HTTP/1.1 200 OK
+ *	{
+ *      "_id": "652c311b6e283244d2ef4c29",
+ *      "userId": "github0000001",
+ *      "status": "ACCEPTED",
+ *      "response": "DECLINED",
+ *      "reviewer": "reviewer1",
+ *      "emailSent": true
+ * 	}
+ *
+ * @apiUse strongVerifyErrors
+ */
+rsvpRouter.put("/", strongJwtVerification, async (req: Request, res: Response) => {
+    const rsvp: boolean | undefined = req.body.isAttending;
+
+    //Returns error if request body has no isAttending parameter
+    if (rsvp === undefined) {
+        return res.status(StatusCode.ClientErrorBadRequest).send({ error: "InvalidParams" });
+    }
+
+    const payload: JwtPayload = res.locals.payload as JwtPayload;
+
+    const userid: string = payload.id;
+
+    const queryResult: DecisionInfo | null = await Models.DecisionInfo.findOne({ userId: userid });
+
+    //Returns error if query is empty
+    if (!queryResult) {
+        return res.status(StatusCode.ClientErrorBadRequest).send({ error: "UserNotFound" });
+    }
+
+    //If the current user has not been accepted, send an error
+    if (queryResult.status != DecisionStatus.ACCEPTED) {
+        return res.status(StatusCode.ClientErrorForbidden).send({ error: "Forbidden" });
+    }
```

Comment:
Change this to "NotAccepted", and modify tests to reflect this - good otherwise!